src/ragger/backend/speculos.py

```diff
@@ -80,6 +80,20 @@ def __init__(self,
     def url(self) -> str:
         return f"http://{self._host}:{self._port}"
 
+    def _retrieve_client_screen_content(self) -> dict:
+        raw_content = self._client.get_current_screen_content()
```

Comment:
Hopefully some OCR improvements should prevent the need for this kind of work around